src/fdk_rdf_parser/rdf_utils/utils.py

```diff
@@ -28,6 +28,19 @@ def object_value(graph: Graph, subject: URIRef, predicate: URIRef) -> Optional[A
     return value.toPython() if value and not isinstance(value, BNode) else None
 
 
+def duration_string_value(
+    graph: Graph, subject: URIRef, predicate: URIRef
+) -> Optional[Any]:
```

Comment:
Den er enten `str(value)` eller `None`?
```suggestion
) -> Optional[str]:
```

Reply:
fix'nd'it